haystack/components/preprocessors/test_chinese_document_spliter.py

```diff
@@ -0,0 +1,366 @@
+"""
+haystack-ai == 2.12.1
+
+"""
+
+from haystack.components.preprocessors import DocumentSplitter
+from copy import deepcopy
+from typing import Any, Callable, Dict, List, Literal, Optional, Tuple
+
+from more_itertools import windowed
+import hanlp
```

Comment:
This should be imported as a lazy import, e.g.:

from haystack.lazy_imports import LazyImport, example:

```python
with LazyImport("Run 'pip install hanlp'") as hanlp_import:
    import hanlp

```